lib/tasks/storage/storage.rb

```diff
@@ -109,7 +109,7 @@ def mirrorable_blobs
   end
 
   def promotable_blobs
-    mirrorable_blobs
+    mirrorable_blobs.where(created_at: (..7.days.ago))
```

Comment:
TIL! Assume this is shorthand for `Time.zone.now..7.days.ago`?

Reply:
Available since https://github.com/mysociety/alaveteli/issues/5224